src/provider/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "experimentalDecorators": true,
-        "emitDecoratorMetadata": true
+        "emitDecoratorMetadata": true,
+        "preserveConstEnums": true
```

Comment:
This doesn't make any behavioural difference, right? Should this be a services-as-a-whole level decision, rather than a project decision?

Reply:
It should't make any major difference.  Docs: https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#94-constant-enum-declarations

I brought up the service wide addition when we added it.  @pjbroadbent seemed on the fence.  I'm OK either way.